src/viam/sdk/tests/mocks/generic_mocks.cpp

```diff
@@ -24,33 +24,13 @@ std::vector<GeometryConfig> MockGeneric::get_geometries(const AttributeMap& extr
     return geometries_;
 }
 std::shared_ptr<MockGeneric> MockGeneric::get_mock_generic() {
-    const auto generic = std::make_shared<MockGeneric>("mock_generic");
+    auto generic = std::make_shared<MockGeneric>("mock_generic");
```

Comment:
Got a weird IDE error without this on the return of `generic` below: `Constness of 'generic' prevents automatic move (clang-tidy performance-no-automatic-move)` .

Reply:
It isn't that weird. It is telling you that it will need to return by copy rather than move, since you can't move from `const`. That's a slight performance penalty, so it is warning you. I think it is a good warning and hopefully clang-tidy will tell us about it.